### PR TITLE
Typos and other imprecisions

### PR DESCRIPTION
**Pull request by daw-dev**
Just take a look at this, it's just some grammar stuff and typos.

### CODE REVIEW

File: AP-protocol.md [hunk 1]
```diff
@@ -354,13 +354,12 @@ pub enum MessageContent {
...
 
 	// Server -> Client
-	RespServerType(ServerType)
+	RespServerType(ServerType),
 	RespFilesList(Vec<u64>),
 	RespFile(Vec<u8>),
 	RespMedia(Vec<u8>),
-	ErrUnsupporedRequestType,
-	ErrRequestedNotFound
-
```
Comment (matteo-dinoia) [inline AP-protocol.md:363]: I would keep the separation line. Between text and media and chat message. But that may just be me.

**Review by matteo-dinoia — APPROVED** [review]
That is fine i guess.

File: AP-protocol.md (hunk 1 above)
Comment (daw-dev) [reply AP-protocol.md:363]: Ok, didn't notice

File: AP-protocol.md [hunk 2]
```diff
@@ -354,13 +354,12 @@ pub enum MessageContent {
...
+	ErrUnsupportedRequestType,
+	ErrRequestedNotFound,
```
Comment (matteo-dinoia) [inline AP-protocol.md:362]:
```suggestion
	ErrRequestedNotFound,

```

**Pull request merged**